app/controllers/game_invitations_controller.rb

```diff
@@ -0,0 +1,65 @@
+class GameInvitationsController < ApplicationController
+
+
+  before_action :require_user, only: %i[join create]
+
+  before_action :load_game, only: %i[create show edit]
+
+  before_action :check_user_is_author?, only: %i[create]
```

Comment:
I think all GM's should be able to regenerate the invite link, not just the author, so I would change this check to `:check_user_is_gm` and also not add the `?` here because it is not a method that answers a question, but a method that executes some code.